spec/forms/steps/address/results_form_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+RSpec.describe Steps::Address::ResultsForm do
+  let(:arguments) { {
+    crime_application: crime_application,
+    record: address_record,
+    lookup_id: lookup_id,
+  } }
+
+  let(:crime_application) { instance_double(CrimeApplication) }
+  let(:address_record) { instance_double(Address, postcode: 'SW1H 0AX') }
+  let(:lookup_id) { '23749191' }
+
+  let(:lookup_service) { instance_double(Ordnance::AddressLookup, call: lookup_results) }
```

Comment:
Haha you might be right. I always default to short and sweet words but in this case it might not make a lot of sense on second thought! 🤣 
Will change it.